app/CMakeLists.txt

```diff
@@ -180,7 +180,7 @@ set(FLATBUFFERS_FLATC_SCHEMA_EXTRA_ARGS
     "--no-union-value-namespacing"
     "--gen-generated"
     "--gen-object-api"
-    "--cpp-ptr-type" "flatbuffers::unique_ptr")
+    "--cpp-ptr-type" "std::unique_ptr")
```

Comment:
There are a few places like this where we switch Flatbuffers over to use std::unique_ptr.